cpp/ql/src/semmle/code/cpp/ir/implementation/raw/internal/TranslatedStmt.qll

```diff
@@ -655,6 +655,10 @@ class TranslatedSwitchStmt extends TranslatedStmt {
       kind = getCaseEdge(switchCase) and
       result = getTranslatedStmt(switchCase).getFirstInstruction()
     )
+    or
+    not stmt.hasDefaultCase() and
+    kind instanceof DefaultEdge and
+    result = getParent().getChildSuccessor(this)
```

Comment:
The `tag` parameter is not bound in this new case, so the default edge is added for every possible instruction tag. I suggest adding `tag = SwitchBranchTag() and`. Same for C#.